packages/dcos-checks/extra/dcos-checks.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DC/OS Poststart Checks: Run node checks
+[Service]
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/dcos-diagnostics.env
+EnvironmentFile=-/opt/mesosphere/etc/dcos-diagnostics-extra.env
+Restart=on-failure
+RestartSec=20
+LimitNOFILE=16384
+PermissionsStartOnly=True
+User=dcos_diagnostics
+# Allow r/w access to the socket file
+SupplementaryGroups=dcos_adminrouter
```

Comment:
Looks like you used another unit as a starting point and some unnecessary stuff got left in. Please make sure this unit file contains only the directives necessary to run checks a single time via the CLI.